advisor/src/test/assets/ort-analyzer-result.yml

```diff
@@ -7,7 +7,7 @@ repository:
     path: "analyzer/src/funTest/assets/projects/synthetic/gradle"
   vcs_processed:
     type: "Git"
-    url: "https://github.com/tests/test-project"
+    url: "https://github.com/tests/test-project.git"
```

Comment:
For consistency, please also align the `vcs_processed` of the projects in this file.

Reply:
You have added `.git.git` in several places.